instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/sqs.py

```diff
@@ -24,3 +29,16 @@ def extract_attributes(self, attributes: _AttributeMapT):
         if queue_url:
             # TODO: update when semantic conventions exist
             attributes["aws.queue_url"] = queue_url
+
+    def on_success(self, span: Span, result: _BotoResultT):
+        operation = self._call_context.operation
+        if operation in SUPPORTED_OPERATIONS:
```

Comment:
Can't we add `messaging.url` too? It's handy in cross-account scenarios.

Reply:
Good idea!